.travis.yml

```diff
@@ -28,12 +29,12 @@ before_script:
   - sudo apt-get install -f -qq
   - sudo dpkg --get-selections | grep hold || { echo "All packages OK."; }
   - sudo apt-get install -q -y cmake-data cmake
-  - sudo apt-get install -qq build-essential
-  - gcc --version && g++ --version # 4.8
+  - sudo apt-get install -qq build-essential g++-5
+  - gcc-5 --version && g++-5 --version # 5.5.0
   - apt-cache search nvidia-*
   - sudo apt-get install -qq nvidia-common
-  - sudo apt-get install -qq nvidia-cuda-dev nvidia-cuda-toolkit # 5.5
-  - sudo apt-get install -qq libboost-dev # 1.54.0
+  - sudo apt-get install -qq nvidia-cuda-dev nvidia-cuda-toolkit # 9.1.85
+  - sudo apt-get install -qq libboost-dev # 1.65.1
```

Comment:
@bernhardmgruber when this PR is merged, please rebase your PR in #169 and remove this line
```suggestion
```